app/script/storage/StorageService.coffee

```diff
@@ -147,16 +156,15 @@ class z.storage.StorageService
       @db.version(3).stores version_3
       @db.version(4).stores version_4
       .upgrade (transaction) =>
-        @logger.warn "Database upgrade to version #{@db.verno}", transaction
-        transaction[@OBJECT_STORE_CLIENTS].toCollection().modify (client) =>
+        @logger.warn 'Database upgrade to version 4', transaction
+        transaction[@OBJECT_STORE_CLIENTS].toCollection().modify (client) ->
           client.meta =
             is_verified: true
             primary_key: 'local_identity'
-          @logger.info 'Updated client', client
       @db.version(5).stores version_4
       @db.version(6).stores version_4
       .upgrade (transaction) =>
-        @logger.warn "Database upgrade to version #{@db.verno}", transaction
+        @logger.warn 'Database upgrade to version 6', transaction
```

Comment:
Doenst work. It will always output the last stores version. Got multiple lines each showing "... to version 11"